cogs/market.py

```diff
@@ -214,12 +214,27 @@ async def remove(self, ctx: commands.Context, id: int):
         if listing["user_id"] != ctx.author.id:
             return await ctx.send("That's not your listing!")
 
+        # confirm
+        pokemon = listing["pokemon"]
```

Comment:
Would need to `Pokemon.build_from_mongo`

Reply:
Sorry, new to this; is it simply `Pokemon.build_from_mongo[listing]` ?

Reply:
It would work to simply delete that line and instead move this line up:

https://github.com/oliver-ni/poketwo/pull/54/files#diff-4fe1e0494898f04d2d06ef3b55fa10863e341514b99533cef07fa87fe8cafb4eR244

Reply:
Ok, updated; see if that works?

Reply:
nice!